src/wiki/achieve.js

```diff
@@ -386,6 +386,25 @@ function featDesc(feat,showFlair){
             wide: true
         });
     }
+    else if (feat === 'existential_risk') {
+        let checked = `<div class="flexed wide">`;    
+        let avcounter = 0;
+        Object.keys(races).sort((a,b) => races[a].name.localeCompare(races[b].name)).forEach(function (key){
+            if (key !== 'protoplasm' && key !== 'nano' && key !== 'synth' && key !== 'junker'){
+                avcounter++;
+                if (global.stats.synth[key]){
+                    checked = checked + `<span class="wide iclr${global.stats.synth[key]}">${races[key].name}</span>`;
+                }
+                else {
+                    checked = checked + `<span class="wide has-text-danger">${races[key].name}</span>`;
+                }
+            }
+        });
+        checked = checked + `</div>`;
+        popover(`f-${feat}`,$(`<div class="wide has-text-label">${feats[feat].desc}</div><div>${loc(`wiki_feat_${feat}`)}</div>${checked}${flair}`),{
+            wide: true
+        });
```

Comment:
This avoids creating the string through concatenation, and instead uses string templates
```js
        const noExistentialRisk = ['protoplasm', 'nano', 'synth', 'junker'];
        const existentialProgress = `<div class="flexed wide">${
            Object.keys(races)
                .filter( (race) => !noExistentialRisk.includes(race) )
                .map( (race) => ({ name: races[race].name, achieve: global.stats.synth[key] }) )
                .sort( (a, b) => a.name.localeCompare(b.name) )
                .map( (race) => `<span class="wide ${
                    race.achieve ? `iclr${race.achieve}` : 'has-text-danger'
                }">${race.name}</span>` )
                .join('')
        }</div>`;
        popover(`f-${feat}`,$(`<div class="wide has-text-label">${feats[feat].desc}</div><div>${loc(`wiki_feat_${feat}`)}</div>${existentialProgress}${flair}`),{
            wide: true
        });
```